nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -114,6 +114,7 @@
         + "max value for max value columns. Properties should be added in the format `initial.maxvalue.<max_value_column>`. This value is only used the first time "
         + "the table is accessed (when a Maximum Value Column is specified). In the case of incoming connections, the value is only used the first time for each table "
         + "specified in the flow files.")
+@PrimaryNodeOnly
```

Comment:
Since GenerateTableFetch can accept incoming flow files, I don't think we should restrict this to run only on the primary node. For example, you could do a ListDatabaseTables, then distribute the flow files among the cluster, where each node's downstream GTF could do the fetch in parallel. In fact that's the main reason we have GTF rather than just QueryDatabaseTable.